methods.py

```diff
@@ -1564,9 +1564,10 @@ def generate_copyright_header(filename: str) -> str:
 /*  %s*/
 /**************************************************************************/
 /*                         This file is part of:                          */
-/*                             GODOT ENGINE                               */
```

Comment:
Please do not change license notification headers, we have no gotten clarification on if doing so or not is correct yet.

Reply:
will revert this in squashed commit